www/js/lib/zimArchive.js

```diff
@@ -115,6 +115,16 @@ define(['zimfile', 'zimDirEntry', 'util', 'utf8'],
         return this._file !== null;
     };
     
+    /**
+     * Looks up the dirEntry's mimetype number in the ZIM file's MIME type list, and returns the corresponding MIME type
+     * 
+     * @param {Integer} mimetype The mimetype number stored in dirEntry.mimetype, used as a lookup value
+     * @return {String} The MIME type corresponding to mimetype in the ZIM file's MIME type list
+     */
+    ZIMArchive.prototype.getMimetype = function(mimetype) {
```

Comment:
In terms of object-oriented programming, I think that resolving this integer to a string should be done in a DirEntry function. Because we have no reason to get a mimetype string from an arbitrary integer (outside of a DirEntry), and a DirEntry already knows its integer.
That's how it is done in libzim.
Each DirEntry knows its zimfile, so it should have access to the `mimeTypes` map.

BTW, we should try to be consistent in the `mimeType` or `mimetype` naming, if it's not complicated

Reply:
OK, I didn't know where to put it. `zimArchive` was a guess, because I reasoned that `zimFile` is for low-level stuff, and getting the MIME type would be a function exposed by the archive itself (as it depends on the archive being loaded). Happy to try putting it in dirEntry.